### PR TITLE
New Workflow: Vulns Remediated Message in Slack

### PR DESCRIPTION
**Pull request by sengleson-r7**
This workflow utilizes the new InsightVM Events Trigger. It triggers on Vulnerability Remediated events, fetches details about each remediated vulnerability, and posts a notification to Slack.

### Checklist
- [x] Workflow verified as working in a test
- [x] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [x] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [x] Workflow includes a detailed Description
- [x] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [x] Triggers, Decisions, and other workflow steps include descriptions where needed
- [ ] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [x] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [x] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [x] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### Screenshots
job
slack_output
workflow1
workflow2

### CODE REVIEW

**Review by cmchugh-r7 — COMMENTED** [review]
should help.md have content? seems to be empty

**Comment by ncramer-r7** [comment]
Hey dropping in a comment here! In workflow.spec.yaml, there are left and right double quotes (“”  vs ""). This can blow up some API requests and our validator doesn't check for that 🙁 (but there's a ticket to create that functionality), do you mind changing it to the standard double quotes? Thanks!

**Review by cmchugh-r7 — APPROVED** [review]

File: workflows/Send_Remediated_Vulnerabilities_Message_in_Slack_from_InsightVM/help.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+* InsightVM
+* InsightConnect
+
+# Documentation
+
+## Setup
+
+*In order to use this workflow, you will need to subscribe to the InsightVM Vulnerabilities Found webhook event. You will also need to adjust the Slack channel name to match your environment.*
+Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.
+Once the workflow has been imported, open the workflow, and view the InsightVM Events trigger step. Click “Save Step” so that you reach the “Instructions” menu. Proceed through the instructions to subscribe to the InsightVM Vulnerabilities Remediated Webhook Event. This will require an Insight Platform User API Key . Insert your User API Key, copy the API request, run it from a command-line interface, and look for a JSON response including an ID. If you receive a JSON response, then your webhook event subscription is active! Close your command-line interface and return to the workflow in InsightConnect.
```
Comment (rzec-r7) [inline workflows/Send_Remediated_Vulnerabilities_Message_in_Slack_from_InsightVM/help.md:27]: I made mention on the other workflow for Teams but is there any benefit in calling out having the user copy and store the response of the registering the webhook incase any of those value might be useful in the future for debug and such?

**Review by rzec-r7 — APPROVED** [review]
other than my one question, looks good

File: workflows/Send_Remediated_Vulnerabilities_Message_in_Slack_from_InsightVM/help.md (hunk 1 above)
Comment (sengleson-r7) [reply workflows/Send_Remediated_Vulnerabilities_Message_in_Slack_from_InsightVM/help.md:27]: sorry Ryan, didn't mean to ignore! We decided not to have users worry about this for now, it isn't a good experience and ultimately the VM team can find registered webhooks by orgID.

**Pull request merged**